commit 33b660dfefe02ecd4391a284f6fd6de9439b2b45
Author: Milian Wolff <mail@milianw.de>
Date:   Thu Mar 20 11:50:07 2014 +0100

    Keep backwards compatibility: replace "dest" variable.
    
    Its an alias for PROJECTDIR and used in quite some templates
    apparently.
    
    BUG: 329392

diff --git a/plugins/appwizard/appwizardplugin.cpp b/plugins/appwizard/appwizardplugin.cpp
index 3e22183..7948e47 100644
--- a/plugins/appwizard/appwizardplugin.cpp
+++ b/plugins/appwizard/appwizardplugin.cpp
@@ -238,6 +238,8 @@ QString AppWizardPlugin::createProject(const ApplicationInfo& info)
     m_variables["APPNAMELC"] = info.name.toLower();
     m_variables["APPNAMEID"] = generateIdentifier(info.name);
     m_variables["PROJECTDIR"] = dest.toLocalFile();
+    // backwards compatibility
+    m_variables["dest"] = m_variables["PROJECTDIR"];
     m_variables["PROJECTDIRNAME"] = dest.fileName();
     m_variables["VERSIONCONTROLPLUGIN"] = info.vcsPluginName;
 
